Assets/Scripts/Quest/NextStageButton.cs

```diff
@@ -32,22 +41,22 @@ public void Update()
         DialogBox db;
         if (Game.Get().quest.phase == Quest.MoMPhase.horror)
         {
-            db = new DialogBox(new Vector2(UIScaler.GetHCenter(-6f), UIScaler.GetBottom(-2.5f)), new Vector2(16, 2), "Horror Phase");
+            db = new DialogBox(new Vector2(UIScaler.GetHCenter(-6f), UIScaler.GetBottom(-2.5f)), new Vector2(16, 2), HORROR_STEP);
```

Comment:
The translations for the phases have some in ALL CAPS And Some In Camel.  I prefer Camel.